src/java.desktop/share/classes/java/awt/font/NumericShaper.java

```diff
@@ -112,25 +112,22 @@
  * <caption>NumericShaper constants precedence</caption>
  * <thead>
  *   <tr>
- *     <th scope="col">Unicode Range
- *     <th scope="col">{@code NumericShaper} Constants
- *     <th scope="col">Precedence
+ *     <th scope="col" id="colheader1">Unicode Range
+ *     <th scope="col" id="colheader2">{@code NumericShaper} Constants
+ *     <th scope="col" id="colheader3">Precedence
  * </thead>
  * <tbody>
  *   <tr>
- *     <th scope="rowgroup" rowspan="2">Arabic
- *     <td>{@link NumericShaper#ARABIC NumericShaper.ARABIC}
+ *     <th scope="rowgroup" rowspan="2" id="rowheader1">Arabic
+ *     <td headers="rowheader1 colheader2">{@link NumericShaper#ARABIC NumericShaper.ARABIC}
  *     <br>
  *     {@link NumericShaper#EASTERN_ARABIC NumericShaper.EASTERN_ARABIC}
- *     <td>{@link NumericShaper#EASTERN_ARABIC NumericShaper.EASTERN_ARABIC}
- *   </tr>
+ *     <td headers="rowheader1 colheader3">{@link NumericShaper#EASTERN_ARABIC NumericShaper.EASTERN_ARABIC}
  *   <tr>
- *     <td>{@link NumericShaper.Range#ARABIC}
+ *     <td headers="rowheader1 colheader2">{@link NumericShaper.Range#ARABIC}
  *     <br>
  *     {@link NumericShaper.Range#EASTERN_ARABIC}
- *     <td>{@link NumericShaper.Range#EASTERN_ARABIC}
- * </tbody>
- * <tbody>
+ *     <td headers="rowheader1 colheader3">{@link NumericShaper.Range#EASTERN_ARABIC}
  *   <tr>
  *     <th scope="row">Tai Tham
```

Comment:
Does it make sense to use `<th scope="rowgroup">` for consistency?

Reply:
Not relevant anymore given last change